### PR TITLE
fix wrong command

### PR DESCRIPTION
**Pull request by BlankSpacePlus**
Install kubectl binary with curl on Linux

English Version:
```shell
curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Chinese Version:
```shell
curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
```

### CODE REVIEW

**Comment by utkarsh-singh1** [comment]
@BlankSpacePlus, This PR seems to involve chnages to multiple language at some time , raise a PR that will introduce changes to english version.

File: content/zh-cn/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ The following methods exist for installing kubectl on Linux:
 
    {{< tabs name="download_binary_linux" >}}
    {{< tab name="x86-64" codelang="bash" >}}
-   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:59]: The indentations are good, why remove them?

Comment (Affan-7) [reply content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:59]: LGTM

IMO it looks good without indentation. The Indentation is annoying when you paste the command into the terminal.

Here are the links for a quick comparison:
https://deploy-preview-43430--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux

https://deploy-preview-43430--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux

Comment (BlankSpacePlus) [reply content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:59]: > The indentations are good, why remove them?



> LGTM
> 
> IMO it looks good without indentation. The Indentation is annoying when you paste the command into the terminal.
> 
> Here are the links for a quick comparison: https://deploy-preview-43430--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux
> 
> https://deploy-preview-43430--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux

Yes, that's what I mean.

**Comment by BlankSpacePlus** [comment]
> This PR seems to involve chnages to multiple language at some time , raise a PR that will introduce changes to english version.

Although this PR involves changes in different natural languages, what I modified is limited to commands and has nothing to do with the natural language type.

File: content/zh-cn/docs/tasks/tools/install-kubectl-linux.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:59]: Well ... looks like we are trapped by the "whole tab snippet" issue again.
Looks like the `tab` doesn't handle indentation well, you have to unindent whatever inside the block. That is annoying.

**Comment by utkarsh-singh1** [comment]
@BlankSpacePlus, many languages have their own guidelines to sync with the upstream (English-version) documentation.

**Comment by utkarsh-singh1** [comment]
You can refer https://github.com/kubernetes/website/pull/43393#issuecomment-1753110245 to find about it more.

File: content/zh-cn/docs/tasks/tools/install-kubectl-linux.md [hunk 2]
```diff
@@ -56,10 +56,10 @@ The following methods exist for installing kubectl on Linux:
...
    {{< /tab >}}
    {{< tab name="ARM64" codelang="bash" >}}
-   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubectl.sha256"
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubectl"
```
Comment (kbhawkey) [inline content/zh-cn/docs/tasks/tools/install-kubectl-linux.md:62]: Hello @BlankSpacePlus 
Besides the removal of the spaces, did you intend to remove the suffix?

**Comment by kbhawkey** [comment]
Hello @BlankSpacePlus .
It's not clear that the changes to the spacing is helpful.
Also, we only accept pull requests changes in one language.

/hold

**Comment by natalisucks** [comment]
@BlankSpacePlus Hi there. For our documentation, we prefer to submit PRs for one language at a time. Can you please amend your PR to only make changes to one language, plus address the feedback left by Karen? Thank you!

**Comment by reylejano** [comment]
@BlankSpacePlus , I'm closing this PR since it's making changes to multiple localizations in one PR and multiple reviewers disagree with some of the proposed changes
/close

**Pull request closed without merging**